e2e/ansible/playbooks/resiliency/test-disk-failure/test-disk-failure.yml

```diff
@@ -0,0 +1,221 @@
+# test-disk-failure.yaml
+# Author: nsathyaseelan
+# Description: Include resiliency test suite in OpenEBS e2e (Test: disk failure)
+
+###############################################################################################
+#Test Steps:
+#1. Copy/Gather Test artifacts to Kubemaster.
+#2. Create Storage pool Path to the additional disk in Nodes.
+#3. Create Storage classes with the newly added storage pool path.
+#4. Deploy Percona application.
+#5. Failing the additional disk in node.
+#6. Verify replica pod status.
+#7. List the scsi hosts and Rescan the disk.
+#8. Get the disk name by using UUID.
+#9. Remount the Disk with the same Mount Point.
+#10 Verify replica pod is scheduled and Running.
+#11 Perform cleanup of test artifacts.
+###############################################################################################
+
+- hosts: localhost
+
+  vars_files:
+    - test-disk-failure-vars.yml
+
+  tasks:
+   - block:
+
+       - name: 1) Install the prerequisites
+         include: test-disk-failure-prereq.yml
+
+       - name: 2) Check status of maya-api server
+         include_tasks: "{{ ansible_env.HOME }}/{{utils_path}}/deploy_check.yml"
+         vars:
+           ns: openebs
+           app: maya-api
```

Comment:
Since we are updating the helm to use stable charts, maya-api server is deprecated and now it will be openeb-api server. Can you make the changes accordingly?